oauth2_http/java/com/google/auth/oauth2/ComputeEngineCredentials.java

```diff
@@ -172,7 +180,8 @@ public GoogleCredentials createScoped(
    * @return new ComputeEngineCredentials
    */
   public static ComputeEngineCredentials create() {
-    return new ComputeEngineCredentials(null, null, null);
+    ComputeEngineCredentials.Builder builder = ComputeEngineCredentials.newBuilder();
```

Comment:
nit: no need to name the builder here (just pass it into the constructor)

Reply:
Agree, updated